README.md

```diff
@@ -95,39 +95,41 @@ Confirm your PHP installation has curl enabled
 
 This quick example assumes that you have a local riak cluster running on port 8098
 
-    require_once('riak-php-client/src/Basho/Riak/Riak.php');
-    require_once('riak-php-client/src/Basho/Riak/Bucket.php');
-    require_once('riak-php-client/src/Basho/Riak/Exception.php');
-    require_once('riak-php-client/src/Basho/Riak/Link.php');
-    require_once('riak-php-client/src/Basho/Riak/MapReduce.php');
-    require_once('riak-php-client/src/Basho/Riak/Object.php');
-    require_once('riak-php-client/src/Basho/Riak/StringIO.php');
-    require_once('riak-php-client/src/Basho/Riak/Utils.php');
-    require_once('riak-php-client/src/Basho/Riak/Link/Phase.php');
-    require_once('riak-php-client/src/Basho/Riak/MapReduce/Phase.php');
-
-    # Connect to Riak
-    $client = new Basho\Riak\Riak('127.0.0.1', 8098);
-
-    # Choose a bucket name
-    $bucket = $client->bucket('test');
-
-    # Supply a key under which to store your data
-    $person = $bucket->newObject('riak_developer_1', array(
-        'name' => "John Smith",
-        'age' => 28,
-        'company' => "Facebook"
-    ));
-
-    # Save the object to Riak
-    $person->store();
-
-    # Fetch the object
-    $person = $bucket->get('riak_developer_1');
-
-    # Update the object
-    $person->data['company'] = "Google";
-    $person->store();
+```php
+require_once 'riak-php-client/src/Basho/Riak/Riak.php';
```

Comment:
So, first and foremost, @Bilge thanks again for your contribution. I have had my head buried into the 2.x rewrite that I have not made many improvements to the old code. I am the official maintainer of this repo and Marco is a community member, who has contributed code to the 1.x version and insight into the 2.x rewrite.

As Marco pointed out, require_once does have an extra validation check in there which is appropriate I think when there is uncertainty in the codebase, but typically require is the better option. Marco was not blaming you for the require_once as it existed long before your PR, but since you had submitted the PR modifying those lines, that we might want to change it now to promote better performance for the users applications who use this lib.

I think what makes the most sense, is that the lib README presents itself in a manor that most PHP devs will be familiar with already. Since composer has been adopted by the PHP community as a whole since the lib was originally started, we are moving towards fully embracing composer. So it makes sense to use the composer autoloader within the README.